stepengine/src/main/java/org/tweetwallfx/devoxx19be/steps/NodeFadeOutStep.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015-2019 TweetWallFX
```

Comment:
FYI: new file should only have the current year
```suggestion
 * Copyright (c) 2019 TweetWallFX
```